README.Rmd

```diff
@@ -0,0 +1,184 @@
+
+## How to "download" the code
+
+In order to keep track on the latest updates of the codes, we suggest not to download the code just once, but to synchronize a local folder with the PACTA_analysis git repository (https://help.github.com/en/github/getting-started-with-github/fork-a-repo). 
+ - Create an user account on github
+ - Fork this repo by clicking the Fork button at the top right  
+ - Clone the repo to your local machine using a git-software (such as github desktop) 
+ - Do frequent updates of the code by your github desktop
+
+## Data Requirements
+The evaluation requires input data that is provided by Asset Resolution (https://asset-resolution.com/). If you are working in a collaboration with the 2DII, the 2DII possibly will allocate the necessary data files as a zip file. Unzip the data and save it a folder, that will later be referred to as data_location 
+
+## Workflow
+
+There are four files that must be run in sequence to create results. 
+
+1_portfolio_check_initialisation.R
+2_project_input_analysis.R
+3_run_analysis.R
+4_report_maker.R
+
+If you stop the process at a certain point (let's say after 2_project_input_analysis), you can go on later at any time with the subsequent steP (with 3_...) but, you after a break/restart of R, you will need to run 1_portfolio_check_initialisation again.
+
+### Initialisation
+
+**1. Set Variables**
+Project wide settings should be set in the file "project_settings.yml", prior to project initialization: 
+
+``` yml
+    project_name: name_of_the_project_folder
+```
+
+   - Please define `project_name`. The project name is relevant for input, output and naming of data files.
+
+``` yml
+    project_internal:
+      twodii_internal: FALSE
+```
+
+   - Ensure that the variable `twodii_internal` is set `FALSE`. Value should only be set to `TRUE` for 2DII internal staff.  
+
+``` yml
+  project_internal:
+    ...
+    project_location_ext: ~/path/to/project
+```
+
+   - Please define ```project_location_ext``` by copying the full path to the folder, in which you want to store data (including the results) related to this particular evaluation. During the run of the code, the a project folder with several sub-folders will be created here.
+   - Please consider: 
+     - copying in a path from windows requires to change all "\" to "/" (e.g. "C:/Users/Desktop/project_folder/)
+     - path needs to be a string ("...")
+
+``` yml
+  project_internal:
+    ...
+    data_location_ext: ~/path/to/data
+```
+
+   - Please define ```data_location_ext``` by copying the full path to the folder, in which you stored the data files (set of rda files) provided Asset Resolution or the 2DII.
+
+   - Please consider: 
+     - copying in a path from windows requires to change all "\" to "/". 
+     - path needs to be a string ("...")
```

Comment:
also pretty sure it automatically turns it into a string, although I don't know if that is true for all setups of course

Reply:
you're right! I left that in from the old README, removing it